lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -68,6 +71,9 @@
     'password': dict(removed_in_version=2.9, no_log=True),
     'ssh_keyfile': dict(removed_in_version=2.9),
 
+    'authorize': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTHORIZE']), type='bool'),
```

Comment:
As this is added in top-level arguments please add `removed_in_version=2.9`

Reply:
We could also not add them here? That seems more reasonable than adding them only to almost immediately take them away.